packages/packages/top-bar/src/components/top-bar.tsx

```diff
@@ -1,35 +1,33 @@
-import { __ } from '@wordpress/i18n';
-import { AppBar, Grid, Box, IconButton, styled } from '@elementor/ui';
-import ElementorIcon from './icons/elementor-icon';
-import PlusIcon from './icons/plus-icon';
-import { openRoute, useIsRouteActive } from '@elementor/v1-adapters';
+import * as React from 'react';
+import { AppBar as BaseAppBar, Box, styled, Grid } from '@elementor/ui';
+import MainMenuLocation from './locations/main-menu-location';
+import ToolsMenuLocation from './locations/tools-menu-location';
+import CanvasViewLocation from './locations/canvas-view-location';
+import UtilitiesMenuLocation from './locations/utilities-menu-location';
+import PrimaryActionLocation from './locations/primary-action-location';
 
-const AppBarAction = styled( IconButton )( ( { theme } ) => ( {
-	borderRadius: '8px',
-	padding: theme.spacing( 2 ),
-	'&:hover, &.active': {
-		backgroundColor: 'rgba(255, 255, 255, 0.1)',
-	},
-} ) );
-
-export const TopBar = () => {
-	const isActive = useIsRouteActive( 'panel/elements' );
+const AppBar = styled( BaseAppBar )`
+	background-color: #232629;
+	height: 48px;
```

Comment:
These values should be read from the theme (height = theme.sizing[ 400 ]).

Reply:
Need to be in another task